hrm-domain/hrm-service/permission-rules/nz.json

```diff
@@ -1,28 +1,26 @@
 {
-  "viewCase": [["isSupervisor"], ["isCreator"]],
-  "closeCase": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "reopenCase": [["isSupervisor"]],
-  "caseStatusTransition": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "addNote": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editNote": [["isSupervisor"]],
-  "addReferral": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editReferral": [["isSupervisor"]],
-  "addHousehold": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editHousehold": [["isSupervisor"]],
-  "addPerpetrator": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editPerpetrator": [["isSupervisor"]],
-  "addIncident": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editIncident": [["isSupervisor"]],
-  "addDocument": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editDocument": [["isSupervisor"]],
-  "editCaseSummary": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editChildIsAtRisk": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "editFollowUpDate": [["isSupervisor"], ["isCreator", "isCaseOpen"]],
-  "viewContact": [["everyone"]],
-  "editContact": [["isSupervisor"], ["isOwner"]],
-  "viewExternalTranscript": [["everyone"]],
-  "viewRecording": [],
-  "viewPostSurvey": [["isSupervisor"]],
-  "viewIdentifiers": [["everyone"]]
-  }
-
+"viewCase":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"closeCase":  [["isSupervisor"]],
+"reopenCase":  [["isSupervisor"]],
+"caseStatusTransition":  [["isSupervisor"]],
+"addNote":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editNote":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editCaseSummary":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editChildIsAtRisk":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editFollowUpDate":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"viewContact":  [["everyone"]],
+"editContact":  [["isSupervisor"],["isOwner"]],
+"viewExternalTranscript":  [["isSupervisor"],["isOwner"],["everyone"]],
+"viewRecording":  [["isSupervisor"],["isOwner"],["everyone"]],
```

Comment:
`everyone` overrides any other statement, they are not needed if you intend to open this for everyone.